### PR TITLE
15663 - update nr result letter

### PR DESCRIPTION
**Pull request by bolyachevets**
*Issue #, [15663](https://github.com/bcgov/entity/issues/15663) [16349](https://github.com/bcgov/entity/issues/16349)

*Description of changes:*

Added the following URLs to web-url vault in 1pass
         DECIDE_BUSINESS_URL = 'https://www.bcregistry.ca/business/auth/home/decide-business'
        BUSINESS_CHANGES_URL = 'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/proprietorships-partnerships/making-changes'
        CORP_FORMS_URL = 'https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/forms-corporate-registry'
        BUSINESS_URL = 'www.bcregistry.ca/business/'

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: api/report-templates/nameRequest.html [hunk 1]
```diff
@@ -19,8 +19,7 @@
                   </div>
                   <div class="report-type-desc">
                       <span class="support-contact-details">
-                         <span class="contact-number">Canada & U.S. Toll Free: 1-877-370-1033</span>
-                        <a href="www.bcregistry.ca/namerequest" target="_blank">www.bcregistry.ca/namerequest</a>
+                           <a href="www.bcregistry.ca/namerequest" target="_blank">www.bcregistry.ca/namerequest</a>
```
Comment (pwei1018) [inline api/report-templates/nameRequest.html:22]: Can this be a variable? We will replace this url soon.

Comment (bolyachevets) [reply api/report-templates/nameRequest.html]: I added NAME_REQUEST_URL to namex vault in 1password, and replaced mentions of www.bcregistry.ca/namerequest with env variable.

Comment (pwei1018) [reply api/report-templates/nameRequest.html]: We already had the URL in web-url vault.

image

Comment (bolyachevets) [reply api/report-templates/nameRequest.html]: ah.. was searching for the old url. i updated vault.json to pull the variable

**Review by pwei1018 — APPROVED** [review]

**Pull request merged**